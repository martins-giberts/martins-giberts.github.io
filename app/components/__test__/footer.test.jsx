/* globals describe, it, expect */

import React from 'react'
import {shallow} from 'enzyme'
import toJson from 'enzyme-to-json'

import Component from '../footer/index'

describe('components/Footer test', () => {
  it('should render without throwing an error', () => {
    const wrapper = shallow(<Component/>)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
