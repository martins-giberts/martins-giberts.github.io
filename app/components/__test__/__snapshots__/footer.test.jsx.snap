// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Footer test should render without throwing an error 1`] = `
<div
  className="footer"
>
  <div
    className="cell"
  >
    Martins Giberts Â© 2017
  </div>
</div>
`;
